pages/recipes/supabase.mdx

```diff
@@ -264,20 +259,25 @@ If you aren't actually needing the data for some of these schemas, you can stop
 
 #### Excluding specific schemas from snapshots
 
-We suggest excluding the `storage` and `supabase_functions` schemas from your snapshots. This is because these schemas contain data that may not have relevant context of the target project you are restoring to.
+We recommand excluding the `storage` and `supabase_functions` schemas from your snapshots. This is because these schemas contain data that may not have relevant context of the target project you are restoring to.
 > e.g: the storage schema may contain files that belong to your production project, but may not be relevant to your development project.
 
 Your snapshot config should look something like this:
 
 ```ts >_&nbsp;snaplet.config.ts
```

Comment:
sorry I missed that earlier, but the `>_&nbsp;` stuff is only needed for bash scripts / commands when you want to represent the terminal, like: `>_&nbsp;terminal`.

Otherwise just use the filename, simply `snaplet.config.ts` here